autodispose/src/main/java/com/uber/autodispose/observers/AutoDisposingSubscriber.java

```diff
@@ -24,4 +25,4 @@
  * A {@link Disposable} {@link Subscriber} that can automatically dispose itself. Interface here
  * for type safety but enforcement is left to the implementation.
  */
-public interface AutoDisposingSubscriber<T> extends Subscriber<T>, Subscription, Disposable {}
+public interface AutoDisposingSubscriber<T> extends FlowableSubscriber<T>, Subscription, Disposable {}
```

Comment:
Nit: would be good to include a tidbit of the explanation you put in the PR description in this commit message